contracts/evm/contracts/xcall/CallService.sol

```diff
@@ -281,9 +281,10 @@ contract CallService is IBSH, ICallService, IFeeManage, Initializable {
 
         string[] memory protocols = req.protocols;
 
-        if (req.messageType == Types.CALL_MESSAGE_TYPE) {
+        if (req.messageType == Types.CALL_MESSAGE_TYPE || req.messageType == Types.PERSISTENT_MESSAGE_TYPE) {
             tryExecuteCall(_reqId, req.to, req.from, _data, protocols);
```

Comment:
Persistent should not try to execute. but execute normally or revert if the execution failed